kardinal-cli/cmd/root.go

```diff
@@ -185,6 +183,18 @@ var dashboardCmd = &cobra.Command{
 	},
 }
 
+var gatewayCmd = &cobra.Command{
+	Use:   "gateway [flow-host]",
```

Comment:
can we actually pass the flow-id instead? we can get host from the flow-id using the `client.GetTenantUuidFlowsWithResponse`